docs/UserGuide.adoc

```diff
@@ -53,6 +53,43 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 
 .  Refer to <<Features>> for details of each command.
 
+== Tutorial
+
+Insurelytics is a Command Line Interface (CLI) application for you to manage your contacts and policies. Here is a quick tutorial to help you get started using the application!
```

Comment:
... for insurance agents like yourself to ...
to answer Who is it for?

Reply:
Edited in the latest commit